edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Kubernetes/edgedeployment/EdgeDeploymentCommand.cs

```diff
@@ -27,6 +27,7 @@ public class EdgeDeploymentCommand : ICommand
         readonly IRuntimeInfo runtimeInfo;
         readonly Lazy<string> id;
         readonly ICombinedConfigProvider<CombinedDockerConfig> combinedConfigProvider;
```

Comment:
```suggestion
        readonly ICombinedConfigProvider<CombinedDockerConfig> dockerConfigProvider;
```

Reply:
I deny this type of changes because code has changed